index.js

```diff
@@ -68,7 +69,8 @@ AFRAME.registerComponent('4dof-controls', {
     }
 
     if (roll !== this.prevRoll) {
-      this.targetEl.setAttribute('position', {z: -this.data.length * roll / this.angleRangeRad});
+      this.zPosition.z = -this.data.length * roll / this.angleRangeRad;
+      this.targetEl.setAttribute('position', this.zPosition);
```

Comment:
Nice catch! Although I'm not sure if there's any recommended way to do that or if `setAttribute('position', 'z', value)` would works better. /cc @dmarcos @ngokevin

Reply:
I tried `setAttribute('position', 'z', value)`, the page wasn't loading... Ah man I think If forgot the quotes on the 'z'. We should try it.

Reply:
After talking with diego he confirmed me that we shouldn't do the three parameters `setAttribute` as it's not a multiparam schema.